test/cli/cli.test.js

```diff
@@ -6,42 +6,7 @@ const internalIp = require('internal-ip');
 const testBin = require('../helpers/test-bin');
 const isWebpack5 = require('../helpers/isWebpack5');
 
-// skip if webpack-dev-server is not linked
-let runCLITest = describe;
-let basePath;
-
-try {
-  basePath = path.join(require.resolve('webpack-dev-server'), '..', '..');
-} catch {
-  runCLITest = describe.skip;
-}
```

Comment:
This'd make tests fail where `webpack-dev-server` is not `npm link`ed.

Reply:
We link it on CI before running tests.

Reply:
But for developers testing locally, it may be a hurdle.